drivers/spi/spi_andes_atcspi200.h

```diff
@@ -11,34 +11,21 @@ LOG_MODULE_REGISTER(spi_atcspi200);
 #include <zephyr/device.h>
 #include <zephyr/drivers/spi.h>
 
-#define REG_IDR         0x00
-#define REG_TFMAT       0x10
-#define REG_DIRIO       0x14
-#define REG_TCTRL       0x20
-#define REG_CMD         0x24
-#define REG_ADDR        0x28
-#define REG_DATA        0x2c
-#define REG_CTRL        0x30
-#define REG_STAT        0x34
-#define REG_INTEN       0x38
-#define REG_INTST       0x3c
-#define REG_TIMIN       0x40
-#define REG_MCTRL       0x50
-#define REG_SLVST       0x60
-#define REG_SDCNT       0x64
-#define REG_CONFIG      0x7c
-
-#define SPI_BASE	(((const struct spi_atcspi200_cfg *)(dev)->config)->base)
-#define SPI_TFMAT(dev)	(SPI_BASE + REG_TFMAT)
-#define SPI_TCTRL(dev)	(SPI_BASE + REG_TCTRL)
-#define SPI_CMD(dev)	(SPI_BASE + REG_CMD)
-#define SPI_DATA(dev)	(SPI_BASE + REG_DATA)
-#define SPI_CTRL(dev)	(SPI_BASE + REG_CTRL)
-#define SPI_STAT(dev)	(SPI_BASE + REG_STAT)
-#define SPI_INTEN(dev)	(SPI_BASE + REG_INTEN)
-#define SPI_INTST(dev)	(SPI_BASE + REG_INTST)
-#define SPI_TIMIN(dev)	(SPI_BASE + REG_TIMIN)
-#define SPI_CONFIG(dev)	(SPI_BASE + REG_CONFIG)
+#ifdef CONFIG_ANDES_SPI_DMA_MODE
+#include <zephyr/dt-bindings/dma/andes_dma.h>
+#include <zephyr/drivers/dma.h>
+#endif
+
+#define SPI_TFMAT(base)		(base + 0x10)
```

Comment:
and keep using them in these macros.

Reply:
OK, no problem.
We have update our source code.
Thank you!